docs/CLI_USAGE.md

```diff
@@ -0,0 +1,30 @@
+# Solution Builder CLI Usage
+
+## Global Variables
+
+### Set and apply a global variable
+
+You can add an anchor to specify where a variable to apply in the solution folder.
+
+For example, in a YAML file:
+```yaml
+detail:
+  PROJECT_ID: old-project-id # sb-var:project_id
+  OTHER: something
+```
+- It sets a variable named "project_id" as the anchor for this "PROJECT_ID" property at the same line in the YAML file.
+
+Then, you can run the following to replace the variable value:
+```
+$ sb vars set project_id new-project-id
```

Comment:
This command is slightly different from existing doc in CSS i.e. `sb set project-id $PROJECT_ID`

Reply:
It's intentional. The `vars` command group will support more things than just project_id.